### PR TITLE
Add `height`, `width` on `symbol`

### PR DESCRIPTION
**Pull request by anthony-redFox**
look at https://developer.mozilla.org/en-US/docs/Web/SVG/Element/symbol

### CODE REVIEW

**Comment by wooorm** [comment]
This project is generated from specs. Did you run the scripts? Otherwise your changes will be immediately undone.

**Comment by anthony-redFox** [comment]
Thanks for suggestion to run script. I add manually attributes because it is bug in spec - https://github.com/w3c/svgwg/issues/803

File: build.js [hunk 1]
```diff
@@ -128,6 +128,9 @@ while (++index < rows2.length) {
   }
 }
 
+// GITHUB ISSUE - Missing attributes in SVGSymbolElement https://github.com/w3c/svgwg/issues/803
```
Comment (wooorm) [inline build.js]:
```suggestion
// Missing from the spec, see: svg-element-attributes#4 <https://github.com/w3c/svgwg/issues/803>
```

**Comment by anthony-redFox** [comment]
Update xo - it is related https://github.com/sindresorhus/eslint-plugin-unicorn/issues/2117

**Comment by wooorm** [comment]
Thanks!

You can run the scripts locally btw: `npm test`.
Now it looks like TS wants `"lib": ["es2021"],` in `tsconfig.json`, or maybe even `es2022`?

**Comment by anthony-redFox** [comment]
done

**Comment by wooorm** [comment]
[released](https://github.com/wooorm/svg-element-attributes/releases/tag/2.1.0), thanks!

**Pull request merged**